utils/duration_test.go

```diff
@@ -0,0 +1,38 @@
+package utils_test
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"go.viam.com/rdk/utils"
+	"go.viam.com/test"
+)
+
+func FuzzDurationJSON(f *testing.F) {
```

Comment:
Can you explain what the fuzz test is doing here? Generating similar test cases to the ones you're `Add`ing? I imagine most fuzz tests result in error?

Reply:
still WIP, but my goal is to demonstrate that for every JSON input - roundtrip (un)marshalling to my custom duration datatype gives the same result as using time.ParseDuration and back.

as I'm doing this I think my whole methodology might be flawed so may backtrack all of this.

Reply:
updated the test with comments on my methodology and I think it's sound now

Reply:
lemme know if this explanation suffices: https://github.com/viamrobotics/rdk/pull/4511/files#r1825152416